zookeeper/src/main/java/com/linecorp/armeria/server/zookeeper/ZooKeeperUpdatingListener.java

```diff
@@ -25,15 +25,20 @@
 import org.apache.zookeeper.CreateMode;
 
 import com.linecorp.armeria.client.Endpoint;
+import com.linecorp.armeria.client.zookeeper.ZooKeeperEndpointGroup;
 import com.linecorp.armeria.common.zookeeper.NodeValueCodec;
 import com.linecorp.armeria.internal.zookeeper.ZooKeeperDefaults;
 import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerListener;
 import com.linecorp.armeria.server.ServerListenerAdapter;
 import com.linecorp.armeria.server.ServerPort;
 
 /**
- * A ZooKeeper Server Listener. When you add this listener, server will be automatically registered
- * into the ZooKeeper.
+ * A {@link ServerListener} which (de)registers the current {@link Server} to
```

Comment:
The following sentence explains deregistration and I think this reads better as just `which registers the current`..., it's the main point of the class.